docs/source/guide/zne-5-theory.myst

```diff
@@ -0,0 +1,110 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+# What is the theory behind ZNE?
+
+
+Zero noise extrapolation (ZNE) {cite}`Giurgica_Tiron_2020_arXiv` is an error
+mitigation technique used to extrapolate the noiseless expectation value of an
+observable from a range of expectation values computed at different noise levels.
+In Mitiq, this process works in two steps:
```

Comment:
```suggestion
This process works in two steps:
```
Better to keep the theory section Mitiq-independent. The aim of this section is to explain what is ZNE in general.